packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss

```diff
@@ -68,12 +68,13 @@ cxl-vaadin-accordion {
       /* stylelint-disable-next-line selector-max-compound-selectors */
       > span {
         display: inline-block;
-        width: 1em;
-        height: 1em;
-        padding: var(--lumo-space-s);
+        width: 1.3em;
+        height: 1.3em;
```

Comment:
We try hard to avoid random numbers.

Let's refactor to `var(--lumo-space-l)` which is the first size that fits 2 digits.